pkg/storage/storage.go

```diff
@@ -64,21 +63,41 @@ func CasList() []func(*client.K8sClient, []string) ([]metav1.TableColumnDefiniti
 func Describe(storages []string, openebsNs, casType string) error {
 	// 1. Create the clientset
 	k, _ := client.NewK8sClient(openebsNs)
-	// 2. Get the namespaces
+	// 2. Get the namespace
 	nsMap, _ := k.GetOpenEBSNamespaceMap()
```

Comment:
(good-first-issue): Implement custom cas-type to namespace mapping logic for entities which are based in `kube-system` namespace.